### PR TITLE
Add --name for icingadb in README.md

### PR DESCRIPTION
**Pull request by pkjam**
The other containers all have meaningful names, describing what they are. Adding this makes it much easier for people to track the purpose of the container.

### CODE REVIEW

**Comment by lippserd** [comment]
@cla-bot check

**Comment by lippserd** [comment]
Thanks for the contribution. We'll have a look at it.

**Review by Al2Klimov — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ docker run --rm -d \
 docker run -d \
 	--network icinga \
 	--restart always \
+        --name icingadb \
 	-e ICINGADB_REDIS_HOST=redis-icingadb \
```
Comment (julianbrost) [inline README.md]: This mixes indentation. The surrounding lines are indented with a tab while the newly added line is indented with spaces (in the other place as well).

**Review by julianbrost — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pkjam) [reply README.md]: Good spot! I've updated the PR with a new commit that resolves this.

File: README.md [hunk 2]
```diff
@@ -56,6 +57,7 @@ Consult the [Icinga DB configuration documentation] on what options there are.
...
```
Comment (julianbrost) [inline README.md]: It's still mixed indentation here.

**Review by julianbrost — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pkjam) [reply README.md]: It should be updated as of commit 1bbcc02
```diff
 	--restart always \
-        --name icingadb \
+	--name icingadb \
 	-e ICINGADB_REDIS_HOST=redis-icingadb \
```

Comment (Al2Klimov) [reply README.md]: 1bbcc02ae22510605890b76429d6119ea7fc986f updates **one of two** changes introduced in 81be45baa2544b60550624efc5ff0445119e6c0c.

Comment (pkjam) [reply README.md]: Ok, I see what you mean.
I just pushed another fix.

File: entrypoint/go.mod [hunk 3]
```diff
@@ -1,10 +1,33 @@
...
+	github.com/mattn/go-colorable v0.1.13 // indirect
+	github.com/mattn/go-isatty v0.0.16 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/redis/go-redis/v9 v9.5.1 // indirect
+	github.com/ssgreg/journald v1.0.0 // indirect
+	go.uber.org/multierr v1.10.0 // indirect
+	golang.org/x/exp v0.0.0-20220613132600-b0d781184e0d // indirect
+	golang.org/x/sync v0.7.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
+	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2 // indirect
```
Comment (Al2Klimov) [inline entrypoint/go.mod]: What's this for?

Comment (pkjam) [reply entrypoint/go.mod]: I think that's because I merged changes from the main branch back to my feature branch.

Comment (Al2Klimov) [reply entrypoint/go.mod]: Please undo.

Comment (pkjam) [reply entrypoint/go.mod]: I force pushed my last good commit. 
So you shouldn't see the golang related diffs any more.

**Review by julianbrost — APPROVED** [review]
Now it's fine. History asks for a squash & merge though.

**Pull request merged**